python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -167,6 +171,43 @@ def test_simple_datasource_read(self) -> None:
             self.assertEqual(len(expectResult), len(actualResult))
 
 
+class ChannelBuilderTests(ReusedPySparkTestCase):
```

Comment:
This should be skipped by `should_test_connect` like `SparkConnectSQLTestCase` in this file.
```
@unittest.skipIf(not should_test_connect, connect_requirement_message)
class SparkConnectSQLTestCase(ReusedPySparkTestCase):
```

I made a PR for that.
- https://github.com/apache/spark/pull/38931/files